src/site/stages/build/drupal/graphql/facilities-fragments/healthCareRegionStaffBios.node.graphql.feature1.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Associated person profiles on the healthcare region node
+ */
+
+const PERSON_PROFILE_RESULTS = `
+  entities {
+    ... on NodePersonProfile {
+      entityPublished
+      flahadfaf
```

Comment:
This is a copy of a file that I'll remove. Just did it so that I could break the query and see that turning the flag on or off worked.